Tool/src/packaging/linux/build_rpm_linux.sh

```diff
@@ -23,7 +23,7 @@ cp ${BGO_SPACE}/THIRD-PARTY-LICENSES.txt ${BGO_SPACE}/bin/linux_amd64/linux/etc/
 echo "Creating the rpm package"
 SPEC_FILE="${BGO_SPACE}/Tool/src/packaging/linux/xray.spec"
 BUILD_ROOT="${BGO_SPACE}/bin/linux_amd64/linux"
-setarch x86_64 rpmbuild --define "rpmversion `cat ${BGO_SPACE}/VERSION`" --define "_topdir bin/linux_amd64/linux/rpmbuild" -bb --buildroot ${BUILD_ROOT} ${SPEC_FILE}
+setarch x86_64 rpmbuild --define "rpmversion ${VERSION}" --define "_topdir bin/linux_amd64/linux/rpmbuild" -bb --buildroot ${BUILD_ROOT} ${SPEC_FILE}
 
 echo "Copying rpm files to bin"
-cp ${BGO_SPACE}/bin/linux_amd64/linux/rpmbuild/RPMS/x86_64/*.rpm ${BGO_SPACE}/build/xray/aws-xray-daemon-`cat ${BGO_SPACE}/VERSION`.rpm
+cp ${BGO_SPACE}/bin/linux_amd64/linux/rpmbuild/RPMS/x86_64/*.rpm ${BGO_SPACE}/build/dist/aws-xray-daemon-linux-amd64-${VERSION}.rpm
```

Comment:
Small observation - didn't see the `.rpm` file in your dist folder screenshot, any idea why?

Reply:
Didn't see arm64 binaries for linux as well.

Reply:
arm64 is there (looks remarkably similar to amd64 ;) ). RPM I can't build on a mac. But the action run also posts a zip with the files can check that out on the PR build status.